test/bijectors/ordered.jl

```diff
@@ -17,20 +17,23 @@ using LinearAlgebra
 end
 
 @testset "ordered" begin
-    d = MvNormal(1:5, Diagonal(6:10))
-    d_ordered = ordered(d)
-    @test d_ordered isa Bijectors.TransformedDistribution
-    @test d_ordered.dist === d
-    @test d_ordered.transform isa OrderedBijector
-    y = randn(5)
-    x = inverse(bijector(d_ordered))(y)
-    @test issorted(x)
+    @testset "$d" for d in [
+        MvNormal(1:5, Diagonal(6:10)),
+        MvTDist(1, 1:5, Matrix(I(5))),
```

Comment:
I think you could use the following alias to fix the failing test:
```suggestion
        Distributions.IsoTDist(1.0, 5, 1.0)
```

Reply:
Yeah that might do the trick, but I'd say that it's a bug that you can't do the above.

Basically, the constructors of MvTDist are specialized to work on Vector ratheer than AbstractVector, which seems overly restrictive.

So I'll raise an issue, and here I'll concretize the range to a vector I think.